ui/DesignSystem/Component/Sidebar.svelte

```diff
@@ -9,6 +9,7 @@
   import AddOrgButton from "./Sidebar/AddOrgButton.svelte";
 
   const dispatch = createEventDispatcher();
+  const session = getContext("session");
```

Comment:
SidebarLayout already depends on session and should extract the information needed and pass it down to the Sidebar component, this will keep this component KISS and data access centralised.